src/System.Console/pkg/System.Console.pkgproj

```diff
@@ -1,9 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
this should be `<Import Project="dir.props" />` now so the new dir.props gets imported.

Reply:
I actually believe that it does import the right one, but will check and make sure that's the case.

Reply:
For the package produced by the System.Console.pkgproj? Huh. I knew the win and unix packages would be correct at least.